src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -23,7 +23,7 @@ import { ChartContentWrapper } from './ChartsHolder.style';
 import { useProjectionsFromRegion } from 'common/utils/model';
 import { LoadingScreen } from 'screens/LocationPage/LocationPage.style';
 import LocationPageHeader from './LocationPageHeader';
-import { summaryToStats } from 'components/NewLocationPage/SummaryStatsBlock/utils';
+import { summaryToStats } from 'components/NewLocationPage/SummaryStat/utils';
```

Comment:
renamed the whole summary stat folder + the files inside, now that its no longer a block // the folder is just the code for the individual stat